javascript/mobile.js

```diff
@@ -0,0 +1,35 @@
+const contactList = document.getElementById('contactList');
+const hambugerButton = document.getElementById('hamburgerButton');
+const hambugerButton2 = document.getElementById('hamburgerButton2');
+const options = document.getElementsByClassName('options');
+
+hambugerButton.addEventListener('click', () => {
+  contactList.classList.toggle('appear');
+  // if(screen.Width > 776)
+  // {
+  //     contactList.style.backgroundImage = "none";
+  // }
```

Comment:
- [ ] please try to remove the unused code in order to keep the file clean and professional